citesphere-model/src/main/java/edu/asu/diging/citesphere/data/bib/impl/CitationMongoDao.java

```diff
@@ -67,6 +67,15 @@ public List<? extends ICitation> findCitationsByContributorUri(List<String> grou
         return mongoTemplate.find(query, Citation.class);
     }
     
+    @Override
+    public long countCitationsByContributorUri(List<String> groupIds, String uri) {
+    	Query query = new Query();
+    	query.addCriteria(Criteria.where("group").in(groupIds));
+        query.addCriteria(Criteria.where("deleted").is(0));
+        query.addCriteria(new Criteria().orOperator(Criteria.where("authors.uri").is(uri), Criteria.where("editors.uri").is(uri), Criteria.where("otherCreators.person.uri").is(uri)));
```

Comment:
duplicate code, the query should be build in an extra method to avoid inconsistencies should the query ever have to be chagned.